resources/dicts/patrols/general/medcat.json

```diff
@@ -479,147 +838,345 @@
+                    },
+                    {
+                        "cats_to": ["s_c"],
+                        "cats_from": ["app1"],
+                        "mutual": true,
+                        "values": ["dislike"],
+                        "amount": 5
+                    }
+                ]
+            },
+            {
+                "text": "At first app1 doesn't speak up, however p_l notices the apprentice being distracted, and gently asked what was wrong. Instead of opening up though, app1 shuts down, refusing to talk about what was bothering {PRONOUN/app1/object} and just says that they should focus on gathering herbs.",
+                "exp": 0,
+                "weight": 20,
+                "herbs": ["random_herbs"],
+                "relationships": [
+                    {
+                        "cats_to": ["app1"],
+                        "cats_from": ["p_l"],
+                        "mutual": false,
+                        "values": ["trust", "comfort"],
+                        "amount": -5
+                    }
+                ]
+            }
+        ]
+    },
+    {
+        "patrol_id": "gen_med_ghost_star1",
+        "biome": ["any"],
+        "season": ["any"],
+        "types": ["herb_gathering"],
+        "tags": [],
+        "patrol_art": "gen_med_ghost_star",
+        "min_cats": 1,
+        "max_cats": 1,
+        "min_max_status": {
+            "medicine cat": [1, 6],
+            "healer cats": [1, 6],
+            "all apprentices": [-1, -1]
+        },
+        "weight": 5,
+        "intro_text": "As p_l pads through c_n territory, {PRONOUN/p_l/subject} {VERB/p_l/feel/feels} a presence join {PRONOUN/p_l/object}.",
+        "decline_text": "{PRONOUN/p_l/poss/CAP} companion stays with {PRONOUN/p_l/object} as silent support while {PRONOUN/p_l/subject} {VERB/p_l/work/works}.",
+        "chance_of_success": 30,
+        "success_outcomes": [
+            {
+                "text": "With a respectful nod, p_l pauses, tucks {PRONOUN/p_l/poss} paws together neatly, and sits to converse with the StarClan cat.",
+                "exp": 30,
+                "weight": 20,
+                "herbs": ["random_herbs"]
+            },
+            {
+                "text": "The scent of a familiar pelt reaches p_l's nose - {PRONOUN/p_l/subject} {VERB/p_l/know/knows} this cat! With a friendly purr, {PRONOUN/p_l/subject} {VERB/p_l/turn/turns} to greet {PRONOUN/p_l/poss} friend from beyond the grave.",
+                "exp": 30,
+                "weight": 5,
+                "herbs": ["random_herbs"]
+            },
             {
                 "text": "With the familiarity brought by {PRONOUN/s_c/poss} connection to StarClan, s_c turns to the ghost haunting {PRONOUN/p_l/poss} path, meeting {PRONOUN/s_c/poss} star-walking ancestor with a purr and quiet confidence.",
                 "exp": 30,
                 "weight": 20,
                 "stat_skill": ["CLAIRVOYANT,2"],
```

Comment:
seems like STAR should also be a valid skill here

Reply:
Good catch! :D Always helps to have more eyes.